src/lava/proc/lif/models.py

```diff
@@ -310,6 +312,49 @@ def spiking_activation(self):
         """
         return self.v > self.effective_vth
 
+@implements(proc=LearningLIF, protocol=LoihiProtocol)
+@requires(CPU)
+@tag('bit_accurate_loihi', 'fixed_pt')
+class PyLearningLifModelBitAcc(PlasticNeuronModelFixed, AbstractPyLifModelFixed):
```

Comment:
Do you need this to be implemented in the lava-nc repo? We dont release BitAcc version right?